lib/python/qmk/cli/list/keymaps.py

```diff
@@ -0,0 +1,23 @@
+"""List the keymaps for a specific keyboard
+"""
+from milc import cli
+import qmk.keymap
+from qmk.errors import NoSuchKeyboardError
+
+@cli.argument("-kb", "--keyboard", help="Specify keyboard name. Example: 1upkeyboards/1up60hse")
+@cli.subcommand("List the keymaps for a specific keyboard")
+def list_keymaps(cli):
+    """List the keymaps for a specific keyboard
+    """
+    # ask for user input if keyboard was not provided in the command line
+    if not cli.config.list_keymaps.keyboard:
```

Comment:
I'd like to recommend using `cli.args.keyboard` here. There's only going to be a single argument for this command, and this syntax is used in several other scripts.

Reply:
Yes, commandline parsing was off. Not both the commandline and the config are handled.